docs/BreakingChanges/080 IL ret not allowed in a try region.md

```diff
@@ -0,0 +1,29 @@
+## 80: IL ret not allowed in a try region
```

Comment:
{
  "Id": "80",
  "Title": "IL ret not allowed in a try region",
  "Details": "Unlike the JIT64 just-in-time compiler, RyuJIT (used in .NET 4.6) does not allow an IL ret instruction in a try region. Returning from a try region is disallowed by the ECMA-335 specification, and no known managed compiler generates such IL. However, the JIT64 compiler will execute such IL if it is generated using reflection emit.",
  "Suggestion": "If an app is generating IL that includes a ret opcode in a try region, the app may target .NET 4.5 to use the old JIT and avoid this break. Alternatively, the generated IL may be updated to return after the try region.",
  "Link": "https://msdn.microsoft.com/en-us/library/dn833125(v=vs.110).aspx#RyuJIT",
  "Markdown": null,
  "ApplicableApis": [
    "Not detectable via API analysis"
  ],
  "Related": null,
  "VersionBroken": "4.6",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": true,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": "We could maybe detect this by looking for apps that emit ret opcodes, but we'd have to also know whether they were emitted into a try block.",
  "IsRetargeting": true,
  "ImpactScope": 3
}

**Parsed APIs:**
- [ ] Not detectable via API analysis